tests/thirdparty/projects.json

```diff
@@ -241,7 +241,26 @@
                     {
                         "dir": "_generate",
                         "commands": [
-                            "go run gen.go -out ../encodeblock_amd64.s -stubs ../encodeblock_amd64.go -pkg s2"
+                            "go generate -v -x"
+                        ]
+                    }
+                ]
+            },
+            {
+                "pkg": "zstd",
+                "module": "_generate/go.mod",
+                "generate": [
+                    {
+                        "dir": "_generate",
+                        "commands": [
+                            "go generate -v -x"
+                        ]
+                    }
+                ],
+                "test": [
+                    {
+                        "commands": [
+                            "go test -short"
```

Comment:
@klauspost Is `-short` sufficient? Without it, the suite runs for approximately 4 minutes.

Reply:
Yes. Should be fine.